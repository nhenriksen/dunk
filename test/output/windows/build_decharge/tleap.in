source leaprc.gaff
source leaprc.water.tip3p
loadamberparams thf.frcmod
THF = loadmol2 thf.mol2
model = combine{ THF THF }
solvateoct model TIP3PBOX 15.0 iso
savepdb model decharge.pdb
saveamberparm model init.decharge.prmtop init.decharge.rst7
quit
