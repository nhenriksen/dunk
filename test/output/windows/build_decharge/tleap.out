-I: Adding /usr/local/amber16/dat/leap/prep to search path.
-I: Adding /usr/local/amber16/dat/leap/lib to search path.
-I: Adding /usr/local/amber16/dat/leap/parm to search path.
-I: Adding /usr/local/amber16/dat/leap/cmd to search path.
-s: Ignoring startup file: leaprc
-f: Source tleap.in.

Welcome to LEaP!
Sourcing: ./tleap.in
----- Source: /usr/local/amber16/dat/leap/cmd/leaprc.gaff
----- Source of /usr/local/amber16/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /usr/local/amber16/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.8, Mar 2015)
----- Source: /usr/local/amber16/dat/leap/cmd/leaprc.water.tip3p
----- Source of /usr/local/amber16/dat/leap/cmd/leaprc.water.tip3p done
Loading library: /usr/local/amber16/dat/leap/lib/atomic_ions.lib
Loading library: /usr/local/amber16/dat/leap/lib/solvents.lib
Loading parameters: /usr/local/amber16/dat/leap/parm/frcmod.tip3p
Reading force field modification type file (frcmod)
Reading title:
This is the additional/replacement parameter set for SPC/E water
Loading parameters: /usr/local/amber16/dat/leap/parm/frcmod.ionsjc_tip3p
Reading force field modification type file (frcmod)
Reading title:
Monovalent ion parameters for Ewald and TIP3P water from Joung & Cheatham JPCB (2008)
Loading parameters: /usr/local/amber16/dat/leap/parm/frcmod.ions234lm_126_tip3p
Reading force field modification type file (frcmod)
Reading title:
Li/Merz ion parameters of divalent to tetravalent ions for TIP3P water model (12-6 normal usage set)
Loading parameters: ./thf.frcmod
Reading force field modification type file (frcmod)
Reading title:
Force field parameters from thf.prmtop
Loading Mol2 file: ./thf.mol2
Reading MOLECULE named THF
Scaling up box by a factor of 1.182645 to meet diagonal cut criterion
  Solute vdw bounding box:              6.619 6.144 5.577
  Total bounding box for atom centers:  42.098 42.098 42.098
      (box expansion for 'iso' is   3.7%)
  Solvent unit box:                     18.774 18.774 18.774
  Volume: 40026.858 A^3 (oct)
  Total mass 20268.080 amu,  Density 0.841 g/cc
  Added 1117 residues.
Writing pdb file: decharge.pdb
   printing CRYST1 record to PDB file with box info
Checking Unit.
Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
 total 0 improper torsions applied
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
  (Residues lacking connect0/connect1 - 
   these don't have chain types marked:

	res	total affected

	THF	2
	WAT	1117
  )
 (no restraints)
	Quit
